cloudmanager/resource_netapp_cloudmanager_connector_gcp.go

```diff
@@ -43,8 +43,17 @@ func resourceOCCMGCP() *schema.Resource {
 			},
 			"service_account_path": {
 				Type:     schema.TypeString,
-				Required: true,
+				Required: false,
```

Comment:
It should be "Optional: true". One of optional, required, or computed must be set true.